Source/SVGKImage.h

```diff
@@ -290,6 +290,9 @@ typedef void (^SVGKImageAsynchronousLoadingDelegate)(SVGKImage* loadedImage, SVG
 /*! returns all the individual CALayer's in the full layer tree, indexed by the SVG identifier of the SVG node that created that layer */
 - (NSDictionary*) dictionaryOfLayers;
 
+/*! returns all the individual CALayer's in the full layer tree as an array */
+- (NSArray*) arrayOfLayers;
```

Comment:
If you can remove that from this PR, I'll merge it - the rest all looks
fine.

If you have a specific problem with dictOfL, point me to an issue / make an
issue and I'll look at it - I use it quite often, and last time I tried it
was working fine. If someone's accidentally broken it, I might be ablt to
spot the problem quickly.

On 16 January 2016 at 20:51, alexandre-g <notifications@github.com> wrote:

> In Source/SVGKImage.h
> <https://github.com/SVGKit/SVGKit/pull/407#discussion_r49937025>:
>
> > @@ -290,6 +290,9 @@ typedef void (^SVGKImageAsynchronousLoadingDelegate)(SVGKImage* loadedImage, SVG
> >  /*! returns all the individual CALayer's in the full layer tree, indexed by the SVG identifier of the SVG node that created that layer */
> >  - (NSDictionary*) dictionaryOfLayers;
> >
> > +/*! returns all the individual CALayer's in the full layer tree as an array */
> > +- (NSArray*) arrayOfLayers;
>
> Thought you might not like that :) To be honest, the reason I've added it
> is because dictionaryOfLayers does not work at the moment, and I wasn't
> able to fix it right off.
> Will submit a new pull request when I do.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/SVGKit/SVGKit/pull/407/files#r49937025>.
>